<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkinSelector</class>
 <widget class="QMainWindow" name="SkinSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Button Mash</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="skinListView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>470</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Regular skins</string>
    </property>
   </widget>
   <widget class="QListView" name="pianoSkinListView">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="pianoCheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Piano display Skin</string>
    </property>
   </widget>
   <widget class="QListView" name="subSkinListView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="previewGraphicView">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>410</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>status label</string>
    </property>
   </widget>
   <widget class="QSplitter" name="configSplitter">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>601</width>
      <height>111</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QPushButton" name="configHSButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Configuration</string>
     </property>
    </widget>
    <widget class="QFrame" name="configFrame">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Source:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sourceLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;SNES Classic&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="changeSourceButton">
          <property name="text">
           <string>Change source</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Skin folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="skinPathEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="skinPathButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
